core/domain/html_cleaner_test.py

```diff
@@ -176,7 +178,7 @@ def test_get_rte_components(self):
             '</oppia-noninteractive-video><br></p>'
         )
 
-        expected_components = [
+        expected_components: Final = [
```

Comment:
This should be of the type `ComponentsDict` right?

Reply:
Yeah, true. I have updated this annotation.